src/renderer/containers/Settings/MeleeOptions.tsx

```diff
@@ -79,6 +86,16 @@ export const MeleeOptions = React.memo(() => {
           <FormControlLabel value={false} label="Launch Dolphin" control={<Radio />} />
         </RadioGroup>
       </SettingItem>
+      <SettingItem name="">
+        <Toggle
+          value={jukeboxEnabled}
+          onChange={(checked) => setJukeboxEnabled(checked)}
+          label="Enable Music"
+          description="Enable music in game via Slippi Jukebox. Incompatible with WASAPI."
+          disabled={disableJukeboxToggle || netplayDolphinOpen}
+          tooltipTitle={disableJukeboxToggle ? "This feature is not yet available" : undefined}
+        />
+      </SettingItem>
```

Comment:
I'm not a big fan of disabling a toggle that they cannot use. Can we just hide/show the toggle based on the Dolphin version?

Reply:
so i was on the fence about that, initially i set it up to show/hide but i wanted to give people an idea of the fact that it is coming, but that also might not be necessary

Reply:
I think ideally this setting shouldn't even be necessary since it should just be normal for a game to have music. But I think it's probably necessary to have because of its incompatible with WASAPI which lot's of pro players use. I also do not think rolling out things that are "coming soon" should really be necessary either.

Reply:
> I think ideally this setting shouldn't even be necessary since it should just be normal for a game to have music.

nah, there has always been some method to turn off music

Reply:
I'll go ahead with the show/hide option